composer.json

```diff
@@ -27,6 +27,7 @@
     "require": {
         "php": ">=7.2",
         "ext-openssl": "*",
+        "ext-apc": "*",
```

Comment:
can we move this to `suggest`  sections ?

Reply:
8e8fb77

Reply:
@Jeckerson

In fact there is a script to install apc(u) on Travis CI: https://github.com/phalcon/cphalcon/blob/b2c72e83c5ed4202075566ba2642c28c6672e32c/.ci/install-php-extensions.sh#L14-L28